src/languages/ini.js

```diff
@@ -54,6 +56,17 @@ export default function(hljs) {
     relevance:0
   };
 
+  var BARE_KEY = /[A-Za-z0-9_-]+/;
+  var QUOTED_KEY_DOUBLE_QUOTE = /"(\\"|[^"])*"/;
+  var QUOTED_KEY_SINGLE_QUOTE = /'[^']*'/;
+  var ANY_KEY = regex.either(
+    BARE_KEY, QUOTED_KEY_DOUBLE_QUOTE, QUOTED_KEY_SINGLE_QUOTE
+  );
+  var DOTTED_KEY = regex.concat(
+    '(?<=^\s*)', ANY_KEY, '(\s*\.\s*', ANY_KEY, ')+',
```

Comment:
Is this look behind? We can't do this because it's not supported everywhere... I guess for now we can just drop this and forget the pinning to the beginning of the line.  I'm not sure it matters.

Reply:
Rest of it looks pretty nice.

Reply:
Yeah right, not worth the breakage for sure. Thanks a lot for your help, I removed the lookbehind and the tests are now passing fine 🙏